ocs_samples/basic_samples/SDS/JavaScript/NodeJs/Sample.js

```diff
@@ -993,32 +1031,9 @@ var app = function (request1, response)
     ).catch(function (err) { logError(err); });  
 
 
-    var getTypesFiltered = printTypes.then(
-        function (res) {
-            if (client.tokenExpires < nowSeconds) {
-                return checkTokenExpired(client).then(
-                    function (res) {
-                        refreshToken(res, client);
-                        return client.getTypes(tenantId, sampleNamespaceId, "Id:*Target*", 0, 100);
-                    }).catch(function (err) { logError(err); });
-            } else {
-                return client.getTypes(tenantId, sampleNamespaceId, "Id:*Target*", 0, 100);
-            }
-        }
-    ).catch(function (err) { logError(err); });
-
-    var printTypesFiltered = getTypesFiltered.then(
-        function (res) {
-            console.log("\nFiltered Types:");
-            console.log(res);
-        }
-    ).catch(function (err) { logError(err); });  
-
-
-
     //tags and metadata
-    var createTags = printTypesFiltered.then( 
-        // Step 15
+    var createTags = printTypes.then( 
```

Comment:
I think the purpose of printing all of the types was to contrast it with the filtered ones. Since we're removing that, this whole step should go away. (I may have missed this in the other 2 samples)